boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_ns.yaml

```diff
@@ -16,3 +16,4 @@ supported:
   - i2c
   - pwm
   - watchdog
+  - netif
```

Comment:
Could use `netif:modem` if we want to be more specific.

Reply:
Done